include/mbgl/perf/trace_data.in

```diff
@@ -0,0 +1,34 @@
+STRUCT_BEGIN(FrameTrace)
+
+STRUCT_MEMBER_UINT64_RANGE(update, true, false)  // Before attaching render target
```

Comment:
I think container with `event-name => event-data` would be sufficient to do the job.

Reply:
We should just use a `const char *` for the name of the event. This will map better with the platform tracing backends.